src/posts/blogs/2019/2019-12-04-layer5-landscape-spectrums-revisited/index.mdx

```diff
@@ -0,0 +1,64 @@
+
+#### 2\. A modular design
+
+The designs I am implementing are meant to be used to present opinionated information. As I am not nearly knowledgeable to form opinionated comparisons between service meshes and related technologies, I have instead made my goal to make it as easy as possible for other Layer5 contributors to utilize the graphics ("\[comparative\] spectrums") that I am designing. Doing so would require encapsulating (templating) the spectrums I design (new and old), within their own HTML files, and providing a means to easily invoke (and customize) instances of those templates into the Landscape page.
+
+#### 3\. Documentation
+
+Because the templates I design are intended to be implemented and customized by other contributors, it is _especially_ critical that those contributors have access to complete and thorough documentation detailing the intended use of each templated design.
+
+* * *
+
+## What I delivered:
+
+#### 1\. A complete toolset
+
+As documented in my [update comment](https://github.com/layer5io/layer5/pull/238#issuecomment-561613644), I designed a table that very closely resembles that of Lee's slides. Implementing the table was interesting, as it ended up requiring [quite a few nested Liquid for-loops](https://github.com/layer5io/layer5/blob/4d396aaa6d3ca46add71531daedc8c0e2a5d2495/_includes/partials/spectrum/table.html#L23). Also as documented, I revamped the grid-based tooling and am quite pleased with the improvements made. The end result of these efforts is a complete set of comparative spectrums.
+
+#### 2, 3. A modular design and documentation
+
+The bulk of the documentation I wrote details how to deal with the modular solution that I designed and implemented to enable comparative spectrums to be included into the Landscape page (through the use of Liquid `include` tags). The documentation speaks for itself; [check it out!](https://github.com/layer5io/layer5/pull/238#issuecomment-561613644)
+
+## What's next?
+
+I would not have considered the documentation I delivered to be complete if I did not report on the issues still present in the work I delivered. I denoted these as 'Current issues' in the documentation, and they represent work that still remains to be done. I know that I can count on the Layer5 community to assist me with small issues like these, and, once this pull request receives some forward motion, I hope to start zeroing in on its smaller issues. Fortunately, I have made plans to continue working with open source well into the new year!
+
+_- [Lee Calcote](https://github.com/leecalcote)_
```

Comment:
This should not be there

Reply:
Thanks for updating this, @Jashpatel1. We do want to include `\- Rafi Unger` in the closing line.